README.md

```diff
@@ -30,14 +30,14 @@ If you only need to use python you can install it directly through pip:
 
     pip install --user git+https://github.com/icecube/nuflux
```

Comment:
Tested with a virtualenv and `CMAKE_PREFIX_PATH=${SROOT} BOOST_ROOT=${SROOT} pip install ./nuflux/`. Works properly with the new branch.